root/admin/wikidoc/WikiDocIndex.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *      /*  ((n   *'.
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {defaults} from 'lodash';
+
+import Layout from '../../layout';
+import Table from '../../components/Table';
+import {withCatalystContext} from '../../context';
+
+import type {WikiDocT} from './types';
+
+type PropsT = {
+  +$c: CatalystContextT,
+  +pages: $ReadOnlyArray<WikiDocT>,
+  +updates_required: boolean,
+  +wiki_server: string,
+  +wiki_unreachable: boolean,
+};
+
+const WikiDocTable = withCatalystContext(({
+  $c,
+  pages,
+  updates_required,
+  wiki_server,
+}: PropsT) => {
+  const columns = React.useMemo(
+    () => {
+      const nameColumn = {
+        Cell: ({cell: {value}}) => (
+          <a href={'/doc/' + encodeURIComponent(value)}>{value}</a>
+        ),
+        Header: N_l('Page name'),
+        accessor: 'id',
+        cellProps: {className: 'title'},
+        headerProps: {className: 'c'},
+        id: 'name',
+      };
+      const transcludedVersionColumn = {
+        Header: N_l('Transcluded version'),
+        accessor: 'version',
+        cellProps: defaults(
+          {className: 'c transcluded-version'},
+          (updates_required && $c.user?.is_wiki_transcluder
+            ? {style: {textAlign: 'right'}}
+            : {}
+          ),
+        ),
+        headerProps: {className: 'c'},
+        id: 'transcluded-version',
+      };
+      const wikiVersionColumn = {
+        Cell: ({row: {original}}) => (
+          <>
+            {original.wiki_version === original.version ? null : (
+              <>
+                <span
+                  className="wiki-version"
+                  style={{color: 'red'}}
+                >
+                  {original.wiki_version === 0
+                    ? l('Error!')
+                    : original.wiki_version}
+                </span>
+                {original.wiki_version ? (
```

Comment:
No since 0 is not a valid wiki version anyway.

Reply:
Isn't that the point? If original.wiki_version === 0 this will create a broken link, no?

Reply:
No, the point is to check `wiki_version` is defined, because it might not be.

Reply:
Ok, I guess diff=0 doesn't lead to a broken link, but I think it's just because it gets read as "the latest version". Given we display 'Error!' just before if it's 0 though, it still seems weird to then go and print a link.

Reply:
There is a misunderstanding, `0` is falsy just like `undefined`, the link won’t be shown in either case.

Reply:
Oh, duh, ok :)